### PR TITLE
3d images pr

### PR DESCRIPTION
**Pull request by shmuelhizmi**

### CODE REVIEW

File: packages/gallery/src/components/helpers/3dManager.ts [hunk 1]
```diff
@@ -0,0 +1,473 @@
...
+        // model.castShadow = true;
+        // model.receiveShadow = true;
+        // scene.add(model);
+        // modelBasePosition = model.position.clone();
+        return {
+          remove() {
+            if (model) {
+              scene.remove(model);
+            }
+          },
```
Comment (shmuelhizmi) [inline packages/gallery/src/components/helpers/3dManager.ts:280]: move thing to CONSTS and remove comments

File: packages/gallery/src/components/item/imageItem.js [hunk 2]
```diff
@@ -62,7 +61,7 @@ class ImageItem extends React.Component {
     return imageContainerClassNames;
   }
 
-  getImageContainer(imageRenderer, classNames, extraNodes) {
+  getImageContainer(imageRenderer, classNames, extraNodes, extraProps = {}) {
```
Comment (shmuelhizmi) [inline packages/gallery/src/components/item/imageItem.js]: make sure this is needed

File: packages/gallery/src/components/item/media/playButton.tsx [hunk 3]
```diff
@@ -3,12 +3,15 @@ import PlayBackground from '../../svgs/components/play_background';
 import PlayTriangle from '../../svgs/components/play_triangle';
 import { clickable } from '../../helpers/mouseCursorPosition';
 
-export const VideoPlayButton = (): JSX.Element => (
+export const VideoPlayButton = (
```
Comment (shmuelhizmi) [inline packages/gallery/src/components/item/media/playButton.tsx]: .

File: packages/playground/src/components/App/App.js [hunk 4]
```diff
@@ -35,7 +35,7 @@ export function App() {
   sideShownOnce = sideShownOnce || showSide;
 
   // const [fullscreenIdx, setFullscreenIdx] = useState(-1);
-  const {numberOfItems = 0, mediaTypes = 'media'} = gallerySettings || {};
+  const {numberOfItems = 0, mediaTypes = 'images'} = gallerySettings || {};
```
Comment (shmuelhizmi) [inline packages/playground/src/components/App/App.js]: revert

File: packages/gallery/src/components/item/itemView.js [hunk 5]
```diff
@@ -449,6 +454,61 @@ class ItemView extends React.Component {
...
+        description={description}
+        id={id}
+        idx={idx}
+        isPrerenderMode={isPrerenderMode}
+        isTransparent={isTransparent}
+        options={options}
+        scene={scene}
+        style={style}
+        settings={settings}
+        isCurrentHover={isCurrentHover}
```
Comment (yhattav) [inline packages/gallery/src/components/item/itemView.js:507]: @shmuelhizmi you have a duplicate prop here

**Pull request merged**